decentraland/metro.config.js

```diff
@@ -1,6 +1,14 @@
 const path = require("path");
 
 module.exports = {
+  // Both lines extracted from: https://medium.com/react-native-training/sharing-code-between-react-web-and-react-native-applications-7f451af26378
+  // Tech-debt: Use same var for the shared dir (Why using resolve instead of join?)
```

Comment:
Having looked at the `path` docs, either `resolve` or `join` is a perfectly sensible approach to achieve the same thing

Reply:
What do you mean by `Use same var for the shared dir`?

Reply:
You've created `var shared = path.join(__dirname, "../shared");`. I've introduced `  watchFolders: [path.resolve(__dirname, "../shared")],`, I've meant to used `shared` variable one `watchFolders` too if is possible.

Reply:
Ah ok, understood!